### DISCUSSION TITLE
Subtraction in EOM

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#57] [Mixtures and flexible systems]
On 2007-11-29 14:22:00, user claudia.leggio wrote:

Hi! What does the constant subtraction mean? If I don't use it in the GAJOE the fit is not good and it is very different to that with the subtraction.
What is that more correct?
Thanks, Claudia

Attachments:

  - [phsave1_146.pdb](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/phsave1_146.pdb)

**Comment by franke-biosaxs** [comment]
On 2007-12-03 17:25:51, user Saxologos wrote:

Constant subtraction means to subtract (or add) a constant number from all experimental data points. The effect of this subtraction is negligible at smaller angles (since the intensity is very high anyway) but gets more important at higher angles. In general it allows for small corrections if the background subtraction was not perfect. However, in the case of flexible or very extended proteins this somehow corrects for the imperfect calculation of the theoretical scattering patterns. You can try to increase the order of spherical harmonics to 50 when running RanCh and see if you can get good fits without using constant subtraction. In general though, if your protein is natively unfolded rather than multidomain with flexible linkers you might need to subtract a constant anyway because of their very high solvation.
In short, if you can get good fits without subtracting a constant, do it.
If the fits are not very good especially at higher angles you can try to see if the constant subtraction improves things.
